core/object/object.h

```diff
@@ -204,7 +264,9 @@ struct PropertyInfo {
 	}
 };
 
+#ifndef DISABLE_DEPRECATED
 TypedArray<Dictionary> convert_property_list(const List<PropertyInfo> *p_list);
```

Comment:
Same question here: this is used in `rendering_server.cpp` in a method that isn't deprecated.